layers/+tools/eaf/config.el

```diff
@@ -0,0 +1,46 @@
+;;; packages.el --- eaf configuration file
+;;
+;; Copyright (c) 2012-2021 Sylvain Benner & Contributors
+;;
+;; Author: Leslie Huang <lesliebinbin19900129@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+(defcustom eaf-apps "Eaf Applications"
```

Comment:
Multiple issues:

1. For `defcustom`, the second positional arg is the default value and the third positional arg is the docstring.
2. The keyword arg `:type` has an invalid value `(symbol)`. At very least, it should be `'symbol`. Here you should use `'(set (const eaf-jupyter) (const eaf-browser) ...)`. See https://www.gnu.org/software/emacs/manual/html_node/elisp/Simple-Types.html
3. If you intended to make this a custom variable, you should not use `defcustom` directly.
Instead you should use `spacemacs|defc`, which is a wrapper of `defcustom`.
4. A better docstring is required. And you also need to document it in layer's README.org.